voicer-client/src/components/marketplace/Marketplace.jsx

```diff
@@ -68,52 +70,16 @@ export default function Marketplace() {
     display = <MultipleJobs data={data} />
   }
   if (jobId !== undefined && jobMatchesDB) {
-    display = <SingleJob data={data} />
+    display = <SingleJob data={data} token={token} history={props.history} />
   }
```

Comment:
do we have a history array?

Reply:
This was working locally, but seems like broke in the netlify deployment. I will refactor to use {withRouter}, which I think is the recommended method of accessing history in non Router-rendered components. Good catch.